deploy.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+if [[ -z $1 ]];
+then
+    printf "DEPLOY.SH:\n\n   usage: ./deploy.sh <local|ic|other> [reinstall]\n\n"
```

Comment:
First of all, thanks for taking the time to share your proposal.

Mind adding the command as a script in the package.json, as we have for the others? Also, add to the documentation `develop/docs/nft-example.md`, how to work with it, a basic description is enough.